README.md

```diff
@@ -1 +1,97 @@
-# Catalog-of-my-things
\ No newline at end of file
+# Catalog-of-my-things
+
+My group created a Ruby console app that displays a catalog of a user.
+
+## Built With
+
+- [x] Ruby (programming language)
+- [x] Linters: [Rubocop](https://rubocop.org/)
+- [x] Visual Studio Code (code editor)
+- [x] RSpec (testings)
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+- [x] A web browser like Google Chrome.
+- [x] A code editor like Visual Studio Code with Git and Ruby.
+
+```
+$ git --version
+```
+
+```
+$ ruby --version && irb
+```
+
+### Setup
+
+Clone the repository using the GitHub link provided below.
+
+### Install
+
+In the terminal, go to your file directory and run this command.
+
+```
```

Comment:
- [ ] Nice work Team, kindly add a video of your presentation where you explain how you code runs and the part you are proud of  here or on the PR the video should be 5 min long maximum :+1:

Reply:
:+1:  Done